change/@fluentui-react-file-type-icons-c16199c3-d573-42c6-85af-8e38a51c0741.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30079&commitId=33512ec70bdb0db809448a143e909985e47901be&env=prod&clientType=FLUENTUI)


  <details><summary><b>PeoplePicker</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PeoplePicker.Normal with text.suggestion-  a .chromium.png|129| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
The diff shows a missing people avatar in a control that's unrelated to the code changed in this PR. Marked as "noise"